tap_airbyte/yarn/main.py

```diff
@@ -127,6 +133,15 @@ def _get_yarn_service_app_id(yarn_config: YarnConfig, service_uri: str) -> str:
     return app_id
 
 
+def delete_yarn_service(yarn_config: YarnConfig, service_name: str) -> None:
+    """
+    Delete a YARN service (this will force remove libraries and files created by the service)
+    """
+    session = _create_session(yarn_config)
+    url = f"{yarn_config.get('base_url')}/app/v1/services/{service_name}"
+    session.delete(url)
```

Comment:
Logs and checks will not be included to avoid Meltano trying to parse it as a message.